Tests/DatadogIntegrationTests/Scenarios/RUM/RUMMobileVitalsScenarioTests.swift

```diff
@@ -0,0 +1,64 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2020 Datadog, Inc.
+ */
+
+import HTTPServerMock
+import XCTest
+
+private extension ExampleApplication {
+    func tapNoOpButton() {
+        buttons["No-op"].tap()
+    }
+
+    func tapBlockMainThreadButton() {
+        buttons["Block Main Thread"].tap()
+    }
+}
+
+class RUMMobileVitalsScenarioTests: IntegrationTests, RUMCommonAsserts {
+    func testRUMMobileVitalsScenario() throws {
+        // Server session recording RUM events send to `HTTPServerMock`.
+        let rumServerSession = server.obtainUniqueRecordingSession()
+
+        let app = ExampleApplication()
+        app.launchWith(
+            testScenarioClassName: "RUMMobileVitalsScenario",
+            serverConfiguration: HTTPServerMockConfiguration(
+                rumEndpoint: rumServerSession.recordingURL
+            )
+        )
+
+        // Wait for the app to settle down for 3 second
+        Thread.sleep(forTimeInterval: 3.0)
+
+        app.tapNoOpButton()
+        app.tapBlockMainThreadButton()
+        app.tapNoOpButton()
+
+        // Get RUM Sessions with expected number of View visits
+        let recordedRUMRequests = try rumServerSession.pullRecordedRequests(timeout: dataDeliveryTimeout) { requests in
+            let visitCount = try RUMSessionMatcher.singleSession(from: requests)?.viewVisits.count
+            return visitCount == 1
+        }
+
+        assertRUM(requests: recordedRUMRequests)
+
+        let session = try XCTUnwrap(RUMSessionMatcher.singleSession(from: recordedRUMRequests))
+
+        XCTAssertEqual(session.viewVisits[0].viewEvents.count, 2)
+        let event1 = session.viewVisits[0].viewEvents[0].view
+        let event2 = session.viewVisits[0].viewEvents[1].view
```

Comment:
As we do:
```
app.tapNoOpButton()
app.tapBlockMainThreadButton()
app.tapNoOpButton()
```
I'd expect that `viewEvents[0]` corresponds to updating the view after sending "application launch" action and `viewEvents[1]` means update after `app.tapNoOpButton()`. As we use both for `longTask.count` comparison, this seems invalid - and might be the reason of CI failure. IMO we should compare `viewEvents[1]` (after `app.tapNoOpButton()`) and `viewEvents[2]` (after `app.tapBlockMainThreadButton()`), no?